### PR TITLE
kafka(ticdc): add kafka go metrics collector and update monitoring panels

### LINKED ISSUE #8340: add kafka-go metrics monitoring

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #8340 

### What is changed and how it works?

* add kafka-go metrics collector.
* adjust some configuration in the kafka-go factory
* add new monitors.
* do not report metrics from DDL sink.

New panels all under the `Sink - MQ Sink` row.

image

#### Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
`None`
```

### CODE REVIEW

**Review by sdojjy — APPROVED** [review]

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -130,8 +130,8 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": 29,
-  "iteration": 1678085403528,
+  "id": null,
```
[the remark this answers is not in the document]
Comment (3AceShowHand) [reply metrics/grafana/ticdc.json:133]: this may set by the grafana automatically, I will fix this manually.

**Comment by 3AceShowHand** [comment]
> Should we mark these metrics only for kafak-go? Then it would not confusing if we use sarama client.

I've add `(Kafka-Go Only)` to related metrics.

**Pull request merged**